### PR TITLE
feat: initial a380_systems modifications

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* This adds the `a380_systems` and `a380_systems` cargo projects, not built automatically.

* [x] Introduce build files
* [x] Integrate #7328 code

To build (in the Linux dev-env):

```
cargo build -p a380_systems_wasm --target wasm32-wasi --release
wasm-opt -O3 -o flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/panel/systems.wasm target/wasm32-wasi/release/a380_systems_wasm.wasm
```

## Testing instructions

N/A

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/systems/a380_systems_wasm/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+version = "0.1.0"
+authors = ["FlyByWire Simulations"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+test = false
+
+[dependencies]
+uom = "0.32.0"
```
Comment (davidwalschots) [inline src/systems/a380_systems_wasm/Cargo.toml:12]:
```suggestion
uom = "0.33.0"
```

**Review by davidwalschots — COMMENTED** [review]
Shall we include those that contributed to the A320's Rust code as co-authors in this? Might be a good thing to do.

File: src/systems/a380_systems_wasm/Cargo.toml (hunk 1 above)
Comment (Benjozork) [reply src/systems/a380_systems_wasm/Cargo.toml]: Done

**Review by crocket63 — APPROVED** [review]
That's just the beginning:)

**Pull request merged**